testsuite/python/dpd.py

```diff
@@ -462,7 +462,7 @@ def test_momentum_conservation(self):
         momentum = np.matmul(s.part[:].v.T, s.part[:].mass)
         for _ in range(10):
             s.integrator.run(25)
-            np.testing.assert_array_less(np.zeros((3, 3)), np.abs(s.part[:].f))
+            np.testing.assert_almost_equal(np.zeros((3,)), np.sum(s.part[:].f))
```

Comment:
shouldn't it be `np.sum(np.abs(s.part[:].f), axis=1)`?

Reply:
Why `abs`? I want to check that the total force is zero.

Reply:
I misunderstood the test logic.